dev_envs/mac-env-setup.md

```diff
@@ -77,6 +77,9 @@ option `iTerm2 > Make iTerm2 Default Term` to make it your default too.
 
 ### Automatic package installation (recommended) ###
 
+NOTE (DEC 2022):  The [CISA `laptop` script](https://github.com/cisagov/laptop/)
```

Comment:
I'd like to see the link to `cisagov/laptop` be turned into a reference-style link since it is used in multiple places.

Reply:
Please see commit 41e2fa9.